tests/load/c/ginkgo_test.go

```diff
@@ -53,21 +63,35 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 })
 
 var _ = ginkgo.Describe("[Load Simulator]", ginkgo.Ordered, func() {
-	var network *tmpnet.Network
+	var (
+		network  *tmpnet.Network
+		tracker  *load.Tracker[common.Hash]
+		logger   = logging.NewLogger("c-chain-load-testing", logging.NewWrappedCore(logging.Info, os.Stdout, logging.Auto.ConsoleEncoder()))
+		tc       = e2e.NewTestContext()
+		r        = require.New(tc)
+		registry = prometheus.NewRegistry()
+	)
+
+	tracker, err := load.NewTracker[common.Hash](registry)
+	r.NoError(err)
 
 	ginkgo.BeforeAll(func() {
 		tc := e2e.NewTestContext()
+		r := require.New(tc)
 		env := e2e.GetEnv(tc)
+
 		network = env.GetNetwork()
 		network.Nodes = network.Nodes[:nodesCount]
 		for _, node := range network.Nodes {
 			err := node.EnsureKeys()
-			require.NoError(tc, err, "ensuring keys for node %s", node.NodeID)
+			r.NoError(err, "ensuring keys for node %s", node.NodeID)
```

Comment:
nit you (we) should prefer using the `require.Function(t` form instead of `r := require.New(t)` since this is error prone to pass the wrong `t`. I had a whole debate with Arran on that which I cannot find, but that's the bits I remember 😄  But feel free to leave unchanged, given avalanchego generally uses this form 🤷